lms/djangoapps/teams/urls.py

```diff
@@ -0,0 +1,41 @@
+"""
+Defines the URL routes for this app.
+"""
+
+from django.conf import settings
+from django.conf.urls import patterns, url
+
+from .views import (
+    TeamsListView,
+    TeamsDetailView,
+    TopicDetailView,
+    TopicListView
+)
+
+TEAM_ID_PATTERN = r'(?P<team_id>[a-z\d_-]+)'
+USERNAME_PATTERN = r'(?P<username>[\w.+-]+)'
```

Comment:
@muzaffaryousaf has created a shared ``USERNAME_PATTERN`` in this PR, but it hasn't merged yet:

https://github.com/edx/edx-platform/pull/8082/files#r31031310

We'll have to be careful to see which one merges first.